KEEP/test-crypto-ivgen-skip-essiv.patch

```diff
@@ -0,0 +1,54 @@
+These tests fail with Illegal instruction and I don't have a clue why,
```

Comment:
i guess we should just disable tests, rather than adding a big patch to fix them